tests/foreman/virtwho/cli/test_esx.py

```diff
@@ -176,6 +175,7 @@ def test_positive_debug_option(self, default_org, form_data, virtwho_config, tar
 
         :CaseImportance: Medium
         """
+        virtwho_config = target_sat.cli.VirtWhoConfig.create(form_data)['general-information']
```

Comment:
> why not use virtwho_config fixture here? and you might need teardown steps if you do not use a fixture

@Gauravtalreja1 This is because the config name has been changed to a new name "new_name = gen_string('alphanumeric')", so the virtwho_config fixture delete virtwho config name is still the old name ,it will throw error.